alacritty/src/input/mod.rs

```diff
@@ -595,8 +595,13 @@ impl<T: EventListener, A: ActionContext<T>> Processor<T, A> {
                 MouseButton::Left => 0,
                 MouseButton::Middle => 1,
                 MouseButton::Right => 2,
-                // Can't properly report more than three buttons..
-                MouseButton::Back | MouseButton::Forward | MouseButton::Other(_) => return,
+                MouseButton::Back => 8,
+                MouseButton::Forward => 9,
+                MouseButton::Other(x) => match x {
+                    6 | 7 => (x + 64) as u8,
+                    10 | 11 => (x + 128) as u8,
```

Comment:
Why did you add this? What is it based on? X11 button IDs? How about cross-platform compatibility?

Reply:
hi

> Why did you add this? What is it based on? X11 button IDs?

it is based on XTerm control sequences (as mentioned here: https://github.com/alacritty/alacritty/pull/6957#issuecomment-1565190532).

> How about cross-platform compatibility?

I think cross-platform compatibility is already being handled by `winit` (ref: https://github.com/rust-windowing/winit/pull/2770)

Reply:
>I think cross-platform compatibility is already being handled by winit (ref: https://github.com/rust-windowing/winit/pull/2770)

This is irrelevant here, because winit guarantee only for `Named` values, but you're mapping `Other` and `Other` is _undefined_. And that's what was the question about here, why you translate `6 | 7` to `x + 64` where on wayland or macOS the buttons are just completely different, thus you kind of implies X11 because xterm is X11.

Reply:
> > I think cross-platform compatibility is already being handled by winit (ref: [rust-windowing/winit#2770](https://github.com/rust-windowing/winit/pull/2770))
> 
> This is irrelevant here, because winit guarantee only for `Named` values, but you're mapping `Other` and `Other` is _undefined_. And that's what was the question about here, why you translate `6 | 7` to `x + 64` where on wayland or macOS the buttons are just completely different, thus you kind of implies X11 because xterm is X11.

alright, I will look into this & update the PR soon.